test-suite/Test.hs

```diff
@@ -0,0 +1,146 @@
+{-# LANGUAGE DataKinds #-}
+{-# OPTIONS_GHC -fno-warn-type-defaults #-}
+import Data.Fixed
```

Comment:
This is just an indication for future reference, but imports (and exports, but they're an unrelated matter) **should** be explicit, see [this](https://wiki.haskell.org/Import_modules_properly).
**Should**, not **must**. It didn't hurt that much to search for where `(@?=)` came from 😄

Reply:
Good catch, it's a habit I'm trying to get into. Thanks!